quiet.js

```diff
@@ -0,0 +1,947 @@
+/* Copyright 2016, Brian Armstrong
+ * quiet.js includes compiled portions from other sources
```

Comment:
why does this get committed and not pulled in as a dependency?

Reply:
quiet.js isn't set up for bundling and, at the time, I didn't feel like debugging why quiet.js was complaining when I had all the files nested in a thirdparty-libs directory. I'm gonna look into other ultrasound libraries now, plus I have to demo this prototype soon, so stuffed them into the root for now. I'll re-examine the ultrasound library placement if quiet.js is kept and this pull request is desired to be merged.